### DISCUSSION TITLE
AutoComplete: add a value slot (like in DropDown)

### DISCUSSION THREAD

**Discussion by Azoorb** [primefaces/community#3192] [PrimeVue]
Currently it is impossible to change a value title (visible text, that user will see when value is selected) via slots unlike DropDown ([link](https://primevue.org/dropdown/#api.dropdown.slots.value))

Is your feature request related to a problem?
I know that i can pass a function into optionLabel prop, but what if i would like to use more complex markup than just text?
Other slots (e.g. option) doesn't have impact when value is selected and pop-up dialog is closed

Describe the solution you'd like
Add value slot like in DropDown component ([link](https://primevue.org/dropdown/#api.dropdown.slots.value))

Describe alternatives you have considered
Alternatively, I can use DropDown with editable text

(from https://github.com/primefaces/primevue/issues/4465)

**Comment by anchan42** [comment]
This would be nice please.
